paddle/fluid/inference/tensorrt/engine.h

```diff
@@ -538,6 +540,17 @@ class TensorRTEngine {
     }
   }
 
+  void getEngineInfo() {
```

Comment:
Baidu's coding style follows [Google Style Guide](http://google.github.io/styleguide/cppguide.html). ([Reference](https://github.com/PaddlePaddle/Paddle/blob/develop/CONTRIBUTING.md#code-style))
So we should use `GetEngineInfo()` instead of `getEngineInfo()` ([Reference](
https://google.github.io/styleguide/cppguide.html#Function_Names))

Reply:
renamed getEngineInfo member function to GetEngineInfo